setup.py

```diff
@@ -64,5 +64,6 @@ def run(self):
     ],
     cmdclass={
         'install': Install
-    }
+    },
+    scripts=["bin/soundconverter"]
```

Comment:
For whatever reason, `bin/soundconverter` didn't get picked by `python setup.py install` on my Fedora until I added this.